library/core/src/ptr/const_ptr.rs

```diff
@@ -1045,8 +1049,9 @@ impl<T: ?Sized> *const T {
     /// leaving the metadata untouched.
     #[must_use]
     #[inline(always)]
-    #[unstable(feature = "pointer_byte_offsets", issue = "96283")]
-    #[rustc_const_unstable(feature = "const_pointer_byte_offsets", issue = "96283")]
+    #[stable(feature = "pointer_byte_offsets", since = "CURRENT_RUSTC_VERSION")]
+    #[rustc_const_stable(feature = "const_pointer_byte_offsets", since = "CURRENT_RUSTC_VERSION")]
+    #[rustc_allow_const_fn_unstable(set_ptr_value)]
```

Comment:
Doesn't seem like @rust-lang/wg-const-eval got pinged for this? This exposes a const-intrinsic on stable, the WG should always be involved when that happens.

Reply:
I can't find any mentions of the WG either, so this looks like an overlook on my part 💀 

We should probably have automation for this kind of thing (i.e. ping the WG if the attributes are getting changed).

Reply:
Yeah that would be nice, but seems hard -- I don't think we have any such automation for any case, currently.

Reply:
Yes, rustbot currently only supports pings based on file names, not diff contents. Although I feel like it shouldn't be too hard to add this functionality :thinking: